parsec/backend/invite.py

```diff
@@ -201,8 +203,16 @@ def generate_invite_email(
             use_ssl=backend_addr.use_ssl,
         )
     )
+    http_invite_link = parsec_invite_link.replace("parsec://", "")
+    split_invite_link = http_invite_link.split("/")
+    http_invite_link = "http://" + split_invite_link[0] + "/api/redirect/" + split_invite_link[1]
     body = [
-        line.format(sender_name=sender_name, invite_link=invite_link, parsec_url=parsec_url)
+        line.format(
+            sender_name=sender_name,
+            http_invite_link=http_invite_link,
+            parsec_invite_link=parsec_invite_link,
```

Comment:
I thin it's better to do the `invite_addr.to_url()` here instead sooner (hence we have access in this function of the addr object which provide useful methods instead of just the string url)